archive_upload/handlers/dsmc_handlers.py

```diff
@@ -653,55 +649,51 @@ def post(self, runfolder):
         path_to_archive_root = self.config["path_to_archive_root"]
         path_to_archive = os.path.abspath(
             os.path.join(path_to_archive_root, runfolder) + "_archive")
-        exclude_dirs = []
-        exclude_extensions = []
 
         # Messages
         invalid_body_msg = "Invalid body format."
         missing_rm_msg = "The `remove` field must be a boolean."
-        exclude_dirs_msg = "The `exclude_dirs` field must be a comma-separated string."
-        exclude_extensions_msg = "The `exclude_extensions` field must be a comma-separated string."
 
         try:
             request_data = json.loads(self.request.body)
         except (ValueError, KeyError):
             raise ArchiveException(reason=invalid_body_msg, status_code=400)
 
-        remove = False
-        if "remove" in request_data:
-            remove = request_data["remove"]
+        remove = request_data.get("remove", False)
```

Comment:
I think a `bool` around the RHS of this line would mean we can skip the if statement below.

Reply:
That didn't quite work (see https://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python) but I think I came up with an ok compromise.

Reply:
Ahh - pheew - good that you noticed that. I should have known better than suggesting that!